server/src/house/house.service.ts

```diff
@@ -36,6 +78,123 @@ export class HouseService {
     }
   }
 
+  async updateHouse(userId: number, houseId: number, dto: UpdateHouseDto) {
```

Comment:
Only admin can do this manipulation.
So should we need the verfication for admin to make sure that this can't be done by users who are not Admin?

Reply:
Thanks! I fixed it
https://github.com/HAK2024/fairy-share/pull/70/commits/a500eec089658f050cb6967ae0dc1c4503d0540f